internal/agent/device/console/session.go

```diff
@@ -37,31 +37,22 @@ func (s *session) close() error {
 }
 
 func (s *session) buildBashCommand(ctx context.Context, metadata *api.DeviceConsoleSessionMetadata) *exec.Cmd {
-	// TODO: add conditional logic for non-bootc devices
-	args := []string{
-		"--ro-bind", "/", "/", // Mount root filesystem read-only (required for bootc)
-		"--bind", "/etc", "/etc", // Provide read/write access to system config files
-		"--bind", "/var", "/var", // Provide read/write access to system state, logs, DBs
-		"--bind", "/sys", "/sys", // Required for block/network interface introspection
-		"--bind", "/run", "/run", // Needed for runtime sockets (D-Bus, systemd, podman)
-		"--bind", "/tmp", "/tmp", // Provide read/write access to /tmp
-		"--bind", "/root", "/root", // Access to root user's home
-		"--dev-bind", "/dev", "/dev", // Provide /dev access: required for journalctl, interactive tty, and system diagnostics
-		"--proc", "/proc", // Mount /proc for ps, top, etc.
-		"--setenv", "SYSTEMD_IGNORE_CHROOT", "1", // Disable chroot detection to enable full systemctl functionality
-		"/bin/bash",
-	}
+	var args []string
 
 	if metadata.TTY {
 		args = append(args, "-i", "-l")
 	}
+
 	if metadata.Command != nil && metadata.Command.Command != "" {
 		args = append(args, "-c", strings.Join(append([]string{metadata.Command.Command}, metadata.Command.Args...), " "))
 	}
-	ret := s.executor.CommandContext(ctx, "bwrap", args...)
+
+	ret := s.executor.CommandContext(ctx, "/bin/bash", args...)
```

Comment:
bubblewrap would be more useful in providing a more restricted env. while we will need this type of thing in the future its overly complicated for now.